### PR TITLE
Set entity monitoring status + fixes

### PR DESCRIPTION
**Pull request by michaelclarkcuadrado**
Adds Set Monitoring Entity Status API and fixes some language here and there.

### CODE REVIEW

**Review by luizfilipe — APPROVED** [review]
LGTM

File: source/index.html.md [hunk 1]
```diff
@@ -1593,16 +1588,51 @@ Type | Description
...
+}'
+```
+
+> The request above returns the following JSON structure:
+
+```json
+{
+    "audit_id": "be73e563-2ece-4daf-b970-f0788db5da98",
+    "entity_urn": "urn:sigma:entity:acaf01af-8797-465c-8f13-9ea6c5baf15d",
+    "monitored": true
```
Comment (olavoholanda) [inline source/index.html.md:1610]: If you use the example of enabled false, the response should be monitored false too

File: source/index.html.md [hunk 2]
```diff
@@ -51,7 +51,7 @@ You must replace <code>"c2lnbWFyYXRpbmdz"</code> with your personal API key.
 When using <code>curl</code> on Microsoft Windows you need to replace single-quotes in the examples with double-quotes. Alternatively, you can replace with <code>\"</code>.
 </aside>
 
-# Checking Endpoints
+# Status Endpoints
```
Comment (olavoholanda) [inline source/index.html.md:54]: Hm, does the audit id check a status endpoint?

**Review by olavoholanda — APPROVED** [review]
Lets merge

**Pull request merged**